apps/music/js/ui/views/player_view.js

```diff
@@ -246,22 +254,18 @@ var PlayerView = {
   },
 
   setCoverImage: function pv_setCoverImage(fileinfo) {
-    // Reset the image to be ready for fade-in
-    this.offscreenImage.src = '';
-    this.coverImage.classList.remove('fadeIn');
-
-    AlbumArtCache.getCoverURL(fileinfo).then(function(url) {
-      this.offscreenImage.addEventListener('load', pv_showImage.bind(this));
-      this.offscreenImage.src = url;
+    LazyLoader.load('js/metadata/album_art_cache.js', function() {
```

Comment:
While you're doing this refactoring, feel free to switch to arrow functions here. It isn't the saved typing that matters, but the fact that you can leave out the bind(this) if you do it.

```
LazyLoader.load('js/metadata/album_art_cache.js', () => {
});
```